### PR TITLE
mossa c4

### PR DESCRIPTION
**Pull request by amagnolo**

### CODE REVIEW

**Comment by mentor-tara** [comment]
### Oh no! You hit a mine. Game over :slightly_frowning_face:

This is the current board:

|   | **A** | **B** | **C** | **D** | **E** | **F** | **G** | **H** |
|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
| **1** |   |   |   |   |   |   |   |   |
| **2** |   |   |   |   |   |   |   |   |
| **3** |   | 2 | 1 | 3 |   |   |   |   |
| **4** |   |   | :bomb: |   |   |   |   |   |
| **5** |   |   |   |   |   |   |   |   |
| **6** |   |   |   |   |   |   |   |   |
| **7** |   |   |   |   |   |   |   |   |
| **8** |   |   |   |   |   |   |   |   |

And this is the revealed board:

|   | **A** | **B** | **C** | **D** | **E** | **F** | **G** | **H** |
|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
| **1** | 1 | :bomb: | 1 | 1 | 2 | :bomb: | 1 | 0 |
| **2** | 1 | 1 | 1 | 1 | :bomb: | 2 | 1 | 0 |
| **3** | 1 | 2 | 1 | 3 | 3 | 3 | 1 | 0 |
| **4** | :bomb: | 3 | :bomb: | 3 | :bomb: | :bomb: | 1 | 0 |
| **5** | 2 | 4 | :bomb: | 3 | 2 | 2 | 1 | 0 |
| **6** | 1 | :bomb: | 2 | 1 | 0 | 0 | 0 | 0 |
| **7** | 1 | 1 | 2 | 1 | 1 | 0 | 0 | 0 |
| **8** | 0 | 0 | 1 | :bomb: | 1 | 0 | 0 | 0 |

---

### This game is over but I created a PR so you can start a new game. You can close this PR.[](clear-c4)

**Comment by amagnolo** [comment]
prova

File: commands.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "commands": [
 "clear c3", 
 "clear d3",
-"clear b3"
+"clear b3",
```
Comment (amagnolo) [inline commands.json:5]: prova commento su file cambiato

File: commands.json [hunk 2]
```diff
@@ -2,6 +2,7 @@
   "commands": [
 "clear c3", 
 "clear d3",
-"clear b3"
```
Comment (amagnolo) [inline commands.json:5]: altro commento su file

**Pull request closed without merging**